### ISSUE TITLE
root-split

### ISSUE THREAD

**Issue by Crashtime** [go-hep/hep#956]
Hello,

I have been trying to use root-split but something seems to be wrong somehow:
$ go install [go-hep.org/x/hep/groot](http://go-hep.org/x/hep/groot)
go: 'go install' requires a version when current directory is not in a
module
     Try 'go install [go-hep.org/x/hep/groot@latest](http://go-hep.org/x/hep/groot@latest)' to install the
latest version
$ go install [go-hep.org/x/hep/groot@latest](http://go-hep.org/x/hep/groot@latest)
go: downloading [go-hep.org/x/hep](http://go-hep.org/x/hep) v0.31.1
go: [go-hep.org/x/hep/groot@latest](http://go-hep.org/x/hep/groot@latest) (in [go-hep.org/x/hep@v0.31.1](http://go-hep.org/x/hep@v0.31.1)):
     The go.mod file for the module providing named packages contains one or
     more replace directives. It must not contain directives that would
cause
     it to be interpreted differently than if it were the main module.

Is this suppose to work? If so is there something I am missing to get it to work?

**Comment by sbinet** [comment]
thanks for the report.

I actually had fixed that in c6ebf251.

let me tag a new version (`v0.32.0`).

in the meantime, this works:

```
$> go install -v go-hep.org/x/hep/groot/cmd/root-split@535619c2
$> $HOME/go/bin/root-split -v
Usage: root-split [options] file.root

ex:
 $> root-split -o out.root -n 10 ./testdata/chain.flat.1.root

options:
  -n int
    	number of events to split into (default 100)
  -o string
    	path to output ROOT files (default "out.root")
  -t string
    	input tree name to split (default "tree")
  -v	enable verbose mode
root-split: missing input file
```

**Comment by sbinet** [comment]
done:

https://go-hep.org/news/release-0.32.0/

apologies for the delay (I wanted to update [Gonum/{gonum,plot}](https://gonum.org) to their latest Go-1.19-compatible version)
